hack/make/install-script

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+set -e
+
+# This script modifies the install.sh script for domains and keys other than
+# those used by the primary opensource releases.
+#
+# You can provide `url`, `yum_url`, `apt_url` and optionally `gpg_fingerprint`
+# or `GPG_KEYID` as environment variables, or the defaults for open source are used.
+#
+# The lower-case variables are substituted into install.sh.
+#
+# If $VERSION is a CS version (e.g. 1.9.1-cs1, 1.9.0-cs1-rc1) you must
+# provide either gpg_fingerprint or GPG_KEYID. If providing GPG_KEYID
+# you must also mount a volume with the correct contents to /root/.gnupg.
+#
+# If $VERSION is not a CS version, gpg_fingerprint and GPG_KEYID are optional,
+# defaulting to the opensource release key ("releasedocker"). Other GPG_KEYIDs
+# will require you to mount a volume with the correct contents to /root/.gnupg.
+#
+# It outputs the modified `install.sh` file to $DOCKER_RELEASE_DIR (default: $DEST)
+#
+# Example usage:
+#
+# docker run \
+# --rm \
+# --privileged \
+# -e "GPG_KEYID=deadbeef" \
+# -e "GNUPGHOME=/root/.gnupg" \
+# -v $HOME/.gnupg:/root/.gnupg \
+# -v $(pwd):/go/src/github.com/docker/docker/bundles \
+# "$IMAGE_DOCKER" \
+# hack/make.sh install-script
+
+: ${DOCKER_RELEASE_DIR:=$DEST}
+: ${GPG_KEYID:=releasedocker}
+
+OSS_URL="https://get.docker.com/"
+OSS_APT_URL="https://apt.dockerproject.org"
+OSS_YUM_URL="https://yum.dockerproject.org"
+OSS_GPG_FINGERPRINT="58118E89F3A912897C070ADBF76221572C52609D"
+
+MAJOR_MINOR_VERSION=$(echo "$VERSION" | awk -F . '{print $1 "." $2;}')
+CS_URL="https://packages.docker.com/$MAJOR_MINOR_VERSION/"
```

Comment:
Yeah, I'd definitely rather we keep our scripts devoid of any CS mention unless it's officially supported by the project, not just the company (with no offense meant -- just a clean separation of concerns).

Overall, I'm definitely +1 to "variablizing" install.sh for greater flexibility while it still lives.